api/views.py

```diff
@@ -1,21 +1,15 @@
 # -*- coding: utf-8 -*-
+from __future__ import division
 from __future__ import print_function
 from __future__ import unicode_literals
-from __future__ import division
 
-from rest_framework.renderers import (JSONRenderer, CoreJSONRenderer)
 from rest_framework.parsers import JSONParser
-from rest_framework_yaml.renderers import YAMLRenderer
-from rest_framework_csv.renderers import CSVRenderer
-from rest_framework_yaml.parsers import YAMLParser
-from rest_framework.views import APIView
-from rest_framework.response import Response
-
-from rest_framework.permissions import AllowAny
+from rest_framework.renderers import (JSONRenderer)
 from rest_framework.response import Response
-from rest_framework.schemas import SchemaGenerator
 from rest_framework.views import APIView
-
+from rest_framework_csv.renderers import CSVRenderer
+from rest_framework_yaml.parsers import YAMLParser
+from rest_framework_yaml.renderers import YAMLRenderer
```

Comment:
Are the CSV and YAML parsers/renderers being used at this time? Just asking, since in this PR you're doing other code cleanups.

Reply:
Yepp, though it's not showing right now on swagger.